tutor/resources/styles/components/top-nav-bar.less

```diff
@@ -1,12 +1,43 @@
-@tutor-navbar-height: 61px;
+@tutor-navbar-height: 60px;
 @tutor-navbar-padding-horizontal: 4rem;
 
 // the @navbar-XX vars are bootstrap defaults
 
 .tutor-top-navbar {
+
+  background: @tutor-white;
+  box-shadow: 0 5px 5px 0 rgba(0, 0, 0, 0.1);
+  color: @openstax-gray;
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  height: @tutor-navbar-height;
+  padding-right: @navbar-padding-horizontal;
+  padding-left:  @navbar-padding-horizontal;
+  z-index: @zindex-navbar-fixed;
+
+  .tutor-nav-controls {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 15px;
+    width: 1200px;
+  }
```

Comment:
Nice idea to have the navbar not expand infinitely.  I think the `1200px` is just a bit too large though, unless maybe the intent is to have it wider than the content?

![screen shot 2017-07-31 at 3 53 11 pm](https://user-images.githubusercontent.com/79566/28797888-6ddb2052-7608-11e7-985c-6dc54ec2e90c.png)
![screen shot 2017-07-31 at 3 51 53 pm](https://user-images.githubusercontent.com/79566/28797889-6dee240e-7608-11e7-84b0-745bd8f2d283.png)